### PR TITLE
[version] bump version to 0.10.10

### PR DESCRIPTION
**Pull request by nerdai**
- confirming that core has been manually bumped/published to v0.10.10

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # ChangeLog
 
+## [0.10.10] - 2024-02-20
+
+I'm still a bit wonky with our publishing process -- apoligies. This is just a version
```
Comment (logan-markewich) [inline CHANGELOG.md:5]: apologies*

**Review by logan-markewich — APPROVED** [review]

**Comment by hatianzhang** [comment]
could you wait a min

**Comment by nerdai** [comment]
> could you wait a min
> 
> Want to add #11064

Aww. I wish I could say yes! I was about to, and then I realized that the version bump has already initiated offline. That is, I have already built/published `llama-index-core` to v0.10.10. This PR completes the version bump by bumping `llama-index` to v0.10.10. 😞

**Comment by hatianzhang** [comment]
ok no worries I can do a post one.

**Pull request merged**